qds_sdk/connection.py

```diff
@@ -43,20 +43,23 @@ def __init__(self, auth, base_url, skip_ssl_cert_check, reuse=True):
             self.session = requests.Session()
             self.session.mount('https://', MyAdapter())
 
-    @retry((RetryWithDelay, requests.Timeout), tries=6, delay=30, backoff=2)
+    @retry((RetryWithDelay, requests.Timeout), tries=12, delay=1, backoff=2, max_delay=32)
     def get_raw(self, path, params=None):
         return self._api_call_raw("GET", path, params=params)
 
-    @retry((RetryWithDelay, requests.Timeout), tries=6, delay=30, backoff=2)
+    @retry((RetryWithDelay, requests.Timeout), tries=12, delay=1, backoff=2, max_delay=32)
     def get(self, path, params=None):
         return self._api_call("GET", path, params=params)
 
+    @retry((RetryWithDelay, requests.Timeout), tries=12, delay=1, backoff=2, max_delay=32)
     def put(self, path, data=None):
         return self._api_call("PUT", path, data)
 
+    @retry((RetryWithDelay, requests.Timeout), tries=12, delay=1, backoff=2, max_delay=32)
```

Comment:
We can't retry on POST because it is not idempotent.
Not sure if we should retry on PUT/DELETE either. Will need to confirm with our rails team that they don't do anything which shouldn't be retried.